core/ide/src/main/java/org/alice/ide/uricontent/FileProjectLoader.java

```
@@ -65,6 +65,10 @@ public URI getUri() {
     return getSaveFile().toURI();
   }
 
+  @Override
+  protected void handleLoadException(File file, Exception e) {
```

Maybe add a comment about the logic of ignoring this exception?